src/styles/blocks/card.scss

```diff
@@ -0,0 +1,155 @@
+@for $index from 1 through $cycle {
+  .main-catalog__card {
+    width: 200px;
```

Comment:
Why do you use loop here? Just style using common classes.

Reply:
Fixed it. Thank you!